README.md

```diff
@@ -122,9 +122,6 @@ Your system should already have most tools, you'll probably only need NML and gi
 Install the developers tools and get NML from the source mentioned above. git is easiest installed via MacPorts:
 `sudo port install git`
 
-On OSX GIMP is not found in the path, if you installed the app package as supplied from the GIMP's project page. You can add that to your search path if you link the binary which requires the X-environment to be running:
-`sudo ln /Applications/Gimp.app/Contents/Resources/bin/gimp /usr/local/bin/gimp`
```

Comment:
Pretty sure OSX people want their hand hold; ask an OSX user where xcftools can be found? (brew? port?)